processor/resourcedetectionprocessor/README.md

```diff
@@ -318,14 +318,18 @@ processors:
 
     * cloud.provider ("aws")
     * cloud.platform ("aws_eks")
+    * k8s.cluster.name
+
+Note: The kubernetes cluster name is only available when running on EC2 instances, and requires permission to run the `EC2:DescribeInstances` [action](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html).
+If you see an error with the message `context deadline exceeded`, please increase the timeout setting in your config.
 
 Example:
 
 ```yaml
 processors:
   resourcedetection/eks:
     detectors: [env, eks]
-    timeout: 2s
+    timeout: 15s
```

Comment:
Is this the avg duration you noticed in your tests? I assume users will be mindful of this additional delay and update readiness check timeouts when the healthcheck extension + k8s probe are enabled.

Reply:
I was generally testing around `15s`, I think around `11s` was the lowest timeout that usually worked before timing out.

Do you think a comment should be added here regarding using this with the healthcheck extension?

Reply:
Just checked the liveness probe values set by default in the helm-chart, we should be good if 15s is the worst observed (probe is run 3 times with 10sec interval before restarting). I was hoping the [healthcheck extension](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/healthcheckextension) had some documentation about things to consider for setting intervals/threshold for health probes, but no dice.